<ui version="4.0" >
 <class>RenderTargetView</class>
 <widget class="QWidget" name="RenderTargetView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="5" >
    <widget class="QFrame" name="m_frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Buffer:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="m_buffer" />
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="m_updateButton" >
     <property name="text" >
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close|QDialogButtonBox::NoButton|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="m_actionNormalSize" >
   <property name="text" >
    <string>1:1</string>
   </property>
   <property name="shortcut" >
    <string>Space</string>
   </property>
  </action>
  <action name="m_actionZoomIn" >
   <property name="text" >
    <string>Zoom In</string>
   </property>
   <property name="shortcut" >
    <string>+</string>
   </property>
  </action>
  <action name="m_actionZoomOut" >
   <property name="text" >
    <string>Zoom Out</string>
   </property>
   <property name="shortcut" >
    <string>-</string>
   </property>
  </action>
  <action name="m_actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="shortcut" >
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_actionClose</sender>
   <signal>triggered()</signal>
   <receiver>RenderTargetView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>255</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
